### PR TITLE
Update docs for functions.php

### PR DESCRIPTION
**Pull request by debabratakarfa**

### CODE REVIEW

**Comment by carolinan** [comment]
Can this be changed to:
Resolve the conflicts.
Only touch the docs and not the spacing/tabs.

**Comment by debabratakarfa** [comment]
Sure @carolinan

**Comment by debabratakarfa** [comment]
Hey @carolinan 
I wish to know, Do I need to follow YODA checking condition or not?

**Comment by ernilambar** [comment]
@debabratakarfa You are fixing lots of things in a single PR. It would be great if modify stuff what PR is supposed to based on the issue created. Another thing, it is good to create issue first and then create PR based on the issue and discussion there. This makes history of git more clean.

**Comment by debabratakarfa** [comment]
Hey @carolinan 

1. Resolve the conflicts. - Done 
2. Only touch the docs and not the spacing/tabs. - Spacing or Tabs as per WPCS.

File: functions.php [hunk 1]
```diff
@@ -507,26 +546,33 @@ function twentytwenty_filter_wp_list_pages_item_classes( $css_class, $item, $dep
...
- * Get and Output Post Meta
- * If it's a single post, output the post meta values specified in the Customizer settings.
- *
- * @param   $post_id int        The ID of the post for which the post meta should be output
- * @param   $location string    Which post meta location to output – single or preview
- */
 if ( ! function_exists( 'twentytwenty_the_post_meta' ) ) :
+	/**
+	 * Get and Output Post Meta
+	 * If it's a single post, output the post meta values specified in the Customizer settings.
```
[the remark this answers is not in the document]
Comment (ntwb) [reply functions.php]: This is still outstanding

**Pull request merged**